scripts/sign_encrypt.py

```diff
@@ -61,8 +64,8 @@ def get_args(logger):
             example offline signing command using OpenSSL:
               base64 -d <UUID>.dig | \\
               openssl pkeyutl -sign -inkey <KEYFILE>.pem \\
-                  -pkeyopt digest:sha256 \\
-                  -pkeyopt rsa_padding_mode:pkcs1 | \\
+                  -pkeyopt digest:sha256 -pkeyopt rsa_pss_saltlen:digest \\
+                  -pkeyopt rsa_padding_mode:pss -pkeyopt rsa_mgf1_md:sha256 |\\
```

Comment:
This help should show both supported algorithms or at least mention that this example matches default `--algo TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256` scheme.

Reply:
This is the sign example with openssl of the default `--algo TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256`. I can add the pkcs1 v1.5 algorithm example if you think it's necessary.

Reply:
I think this should at least state that this example covers the default algo. I.e.:
```patch
               The command \'digest\' can be aliased by ''' + dat + '''
               The command \'stitch\' can be aliased by ''' + sat + '\n' + '''
-            example offline signing command using OpenSSL:
+            example of offline signing command using OpenSSL for default algo \\ 
+            TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256:
               base64 -d <UUID>.dig | \\
               openssl pkeyutl -sign -inkey <KEYFILE>.pem \\
```